x-pack/test/observability_functional/apps/observability/alerts/add_to_case.ts

```diff
@@ -67,30 +67,5 @@ export default ({ getService, getPageObjects }: FtrProviderContext) => {
         });
       });
     });
-
-    describe.skip('When user has read permissions for cases', () => {
```

Comment:
@fkanout Why did you remove these tests? They are not related to workflow status, but they are related to the `add to case` options. I think they should stay.

Reply:
I need to double-check that, it was skipped and it uses `openActionsMenuForRow`.